pkg/eventmonitor/interface.go

```diff
@@ -13,8 +13,8 @@ type EventConsumer interface {
 	probe.EventConsumerInterface
 }
 
-// EventConsumerInterface defines an event consumer
-type EventConsumerInterface interface {
+// Module defines an event monitor module
+type Module interface {
```

Comment:
My only concern is this will be confused with system-probe level modules.

Reply:
I'm not convinced neither, I'll try to find a better name (even I didn't until now :)